pkg/render/dex_config.go

```diff
@@ -458,7 +458,7 @@ func (d *dexConfig) Connector() map[string]interface{} {
 	}
 
 	//Google specific.
-	if d.idpSecret.Data[serviceAccountSecretField] != nil || d.idpSecret.Data[adminEmailSecretField] == nil {
+	if d.idpSecret.Data[serviceAccountSecretField] != nil && d.idpSecret.Data[adminEmailSecretField] != nil {
 		config["serviceAccountFilePath"] = serviceAccountSecretLocation
```

Comment:
Minor: Any reason `"serviceAccountFilePath"` is hard-coded string instead of a constant like the other values? 

The string is only used once, so there's that. But, it still feels more consistent to have it defined in a const like the others.